cesql/spec.md

```diff
@@ -229,38 +259,35 @@ Corresponds to the syntactic rule `unary-operation`:
+| `x AND y: Boolean x Boolean -> Boolean` | Returns the logical and of `x` and `y`                                                                                             |
+| `x OR y: Boolean x Boolean -> Boolean`  | Returns the logical or of `x` and `y`                                                                                              |
+| `x XOR y: Boolean x Boolean -> Boolean` | Returns the logical xor of `x` and `y`                                                                                             |
+| `x = y: Integer x Integer -> Boolean`   | Returns `true` if the values of `x` and `y` are equal                                                                              |
+| `x != y: Integer x Integer -> Boolean`  | Same as `NOT (x = y)`                                                                                                              |
+| `x <> y: Integer x Integer -> Boolean`  | Same as `NOT (x = y)`                                                                                                              |
+| `x < y: Integer x Integer -> Boolean`   | Returns `true` if `x` is less than `y`                                                                                             |
+| `x <= y: Integer x Integer -> Boolean`  | Returns `true` if `x` is less than or equal to `y`                                                                                 |
+| `x > y: Integer x Integer -> Boolean`   | Returns `true` if `x` is greater than `y`                                                                                          |
+| `x >= y: Integer x Integer -> Boolean`  | Returns `true` if `x` is greater than or equal to `y`                                                                              |
+| `x * y: Integer x Integer -> Integer`   | Returns the product of `x` and `y`                                                                                                 |
+| `x / y: Integer x Integer -> Integer`   | Returns the result of dividing `x` by `y`, rounded towards `0` to obtain an integer. Returns `0` and a _MathError_ if `y = 0`      |
+| `x % y: Integer x Integer -> Integer`   | Returns the remainder of `x` divided by `y`, where the result has the same sign as `x`. Returns `0` and a _MathError_ if `y = 0`   |
+| `x + y: Integer x Integer -> Integer`   | Returns the sum of `x` and `y`                                                                                                     |
+| `x - y: Integer x Integer -> Integer`   | Returns the difference of `x` and `y`                                                                                              |
+| `x = y: String x String -> Boolean`     | Returns `true` if the values of `x` and `y` are equal                                                                              |
```

Comment:
For easier reading, I'd put something like `(case sensitive)` on each of these lines so that people don't need to look outside of each row to get the full definition of the operator. Then you can delete the sentence you added on line 289.